apps/examples/testcase/ta_tc/task_manager/itc/itc_taskmanager_main.c

```diff
@@ -672,63 +744,51 @@ void itc_taskmanager_group_unregister_p(void)
 	ret = task_manager_start(tm_sample_handle3, TM_RESPONSE_WAIT_INF);
 	TC_ASSERT_EQ_CLEANUP("task_manager_start", ret, OK, unregister_tasks_4(sample_info->tm_gid));
 
-	usleep(100);
+	usleep(1);
 
-	unregister_tasks_4(sample_info->tm_gid);
+	int status;
```

Comment:
Format like below could be followed:

<Declarations>
<Test case>
<Unregister>
<Success Result>

Reply:
fixed it